bindings/java/src/test/java/org/apache/opendal/StepsTest.java

```diff
@@ -0,0 +1,58 @@
+package org.apache.opendal;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StepsTest {
+
+    Operator operator;
+
+    @Given("A new OpenDAL Blocking Operator")
+    public void a_new_open_dal_blocking_operator() {
+        Map<String, String> params = new HashMap<>();
+        params.put("root", "/tmp");
+        this.operator = new Operator("Memory", params);
+    }
+
+    @When("Blocking write path \"test\" with content \"Hello, World!\"")
+    public void blocking_write_path_test_with_content_hello_world() {
+        this.operator.write("test", "Hello, World!");
+    }
+
+
+    @Then("The blocking file \"test\" should exist")
+    public void the_blocking_file_test_should_exist() {
+
+    }
+
+
+    @Then("The blocking file \"test\" entry mode must be file")
+    public void the_blocking_file_test_entry_mode_must_be_file() {
+        // Write code here that turns the phrase above into concrete actions
+//        throw new io.cucumber.java.PendingException();
+    }
+
+    @Then("The blocking file \"test\" content length must be 13")
+    public void the_blocking_file_test_content_length_must_be_13() {
+        // Write code here that turns the phrase above into concrete actions
+        String content = this.operator.read("test");
```

Comment:
Does `cucumber` support capturing the value we input? It would be better to refer to a variable instead of manually writing `"test"` here.

For example:

https://github.com/apache/incubator-opendal/blob/85c3785d5ff2543278325788916da43fe2dc91e9/bindings/python/tests/steps/binding.py#L38-L50

Reply:
We should use the var which is injected.

https://cucumber.io/docs/cucumber/step-definitions/?lang=java